### PR TITLE
extracted login form from login page

### PR DESCRIPTION
**Pull request by paulmatchett**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Nice, looks a lot cleaner to me. I'd wait for feedback from the others as well.

**Review by ghost — COMMENTED** [review]
Reverting my approval, until I see the finished product

**Review by ghost — CHANGES_REQUESTED** [review]
changed

File: web-app/src/components/LoginForm/index.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          required: true,
+          isEmail: true,
+        }}
+      />
+      <Input
+        type="input"
+        htmlAttrs={{
+          type: 'password',
+          name: 'password',
+          placeholder: 'Enter an password',
```
Comment (d1wilko) [inline web-app/src/components/LoginForm/index.jsx]: I think this should be 'Enter **a** password'

File: web-app/src/components/LoginForm/container.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Swal from 'sweetalert2';
+import { PropTypes as PT } from 'prop-types';
+import { userLogin } from '../../services/userService';
+
+export default Wrapped =>
```
Comment (d1wilko) [inline web-app/src/components/LoginForm/container.js:6]: Can we name this something that better explains what it is? `Wrapped` is a bit abstract

**Review by d1wilko — CHANGES_REQUESTED** [review]
Some real good stuff, 2 suggestions

File: web-app/src/components/LoginForm/container.js (hunk 2 above)
Comment (paulmatchett) [reply web-app/src/components/LoginForm/container.js:6]: Wrapped is what has been used everywhere in the project that has this pattern. I was just following the convention.

Comment (d1wilko) [reply web-app/src/components/LoginForm/container.js:6]: fair enough, not a big fan of it but it's cool

File: web-app/src/components/LoginForm/index.jsx (hunk 1 above)
Comment (paulmatchett) [reply web-app/src/components/LoginForm/index.jsx]: typo corrected

File: web-app/src/components/LoginForm/container.js (hunk 2 above)
Comment (paulmatchett) [reply web-app/src/components/LoginForm/container.js:6]: I agree. I'll put it to the group and see if we can make it a fix throughout the project on another pull request.

File: web-app/src/components/LoginForm/index.jsx (hunk 1 above)
Comment (d1wilko) [reply web-app/src/components/LoginForm/index.jsx]: can you push the change?

**Review by d1wilko — APPROVED** [review]
looks good to me

**Review by BrownMark — APPROVED** [review]

**Pull request merged**